src/lib/components/progressbar/ProgressBar.component.js

```diff
@@ -0,0 +1,235 @@
+//@flow
+import React, { useEffect, useState } from 'react';
+import styled, { css } from "styled-components";
+
+export type ProgressBarProps = {
+  total: number,
+  filled: number,
+  size?: Size,
+  color: string,
+  measure: string,
+  percentage?: boolean,
+  capacity?: boolean,
+  label?: boolean
+};
+
+export const ProgressBarContainer = styled.div`
+  display: block;
+  background: #d9d9d9;
+
+  ${props => {
+    switch (props.size) {
+      case "smaller":
+        return css`
+          border-radius: 12px;
+          width: 200px;
+          height: 10px;
+        `;
+
+      case "small":
+        return css`
+          border-radius: 12px;
+          width: 250px;
+          height: 15px;
+        `;
+
+      case "large":
+        return css`
+          border-radius: 12px;
+          width: 300px;
+          height: 15px;
+        `;
+
+      case "larger":
+        return css`
+          border-radius: 12px;
+          width: 400px;
+          height: 20px;
+        `;
+
+      default:
+        return css`
+          border-radius: 12px;
+          width: 200px;
+          height: 10px;
+        `;
+  }
+}}
+`;
+
+export const Container = styled.div`
+  .percentage {
+    font-size: 20px;
+    display: inline-block;
+    font-weight: 800;
+    margin-bottom: 4px;
+    width: 50%;
+    text-align: left;
+  }
+
+  .capacity {
+    display: inline-block;
+    width: 50%;
+    text-align: right;
+    font-size: 14px;
+    color: #908d8d;
+  }
+
+  .used {
+    display: inline-block;
+    width: 50%;
+    text-align: left;
+  }
+
+  .free {
+    display: inline-block;
+    width: 50%;
+    text-align: right;
+  }
+
+  margin: 10px 0 10px 0;
+
+  ${props => {
```

Comment:
idem here, you do not need set the width. We need to allow users to set the width they want to. 
In the storybook you can set the width like below:
```
<div style= {{width: 200px}}>
    <ProgressBar .../>
</div>
```